packages/components/src/styles/components/flyout.scss

```diff
@@ -27,6 +29,10 @@
   border: none;
   box-shadow: 0 2px 3px 0 #3b3d4540, 0 12px 24px 0 #3b3d4559; // TODO: convert to token
 
+  &:not([open]) {
```

Comment:
I would expect this to come _after_ `&[open]`

Reply:
Cleaning this up as part of related refactors, thanks